auditlogsreceiver/audit_logs.go

```diff
@@ -31,16 +31,21 @@ type auditLogsReceiver struct {
 	logger       *zap.Logger
 	pollInterval time.Duration
 	pageLimit    int
-	wg           *sync.WaitGroup
-	doneChan     chan bool
-	storage      storage.Storage
-	rest         *resty.Client
-	consumer     consumer.Logs
+
+	wg     *sync.WaitGroup
+	cancel context.CancelFunc
```

Comment:
What do you think about using `stopPolling` name for this closure? We have `startPolling` so it would be symmetrical ;)

Reply:
Done.